apps/barista/src/components/color-component-wrapper/color-wrapper.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../styles/global/variables';
```

Comment:
I would really like to reuse this styles and not duplicate them. Is there a way to use the color-grid for both usecases? I.e. if there is a color-input given, we generate the `<ba-color>` items, otherwise we use the given `<ng-content>`.

@thomaspink what do you think? maybe let's discuss this tomorrow :)